frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/selectors/immutable/wizard-selectors.ts

```diff
@@ -55,12 +60,39 @@ export const getStepsMetadata = (state, wizardID: string): VMWizardTabsMetadata
     delete result[tab].value;
   });
 
+  const isProviderImport = iGetCommonData(state, wizardID, VMWizardProps.isProviderImport);
+  const isSimpleView = iGetCommonData(state, wizardID, VMWizardProps.isSimpleView);
+
+  const visibleTabls = ALL_VM_WIZARD_TABS.filter((tab) => !result[tab].isHidden);
+  const canJumpToAll =
+    result[VMWizardTab.RESULT].isLocked ||
+    result[VMWizardTab.RESULT].isValid ||
+    result[VMWizardTab.RESULT].isPending;
+  const isLocked = _.some(visibleTabls, (tab) => result[tab]?.isLocked);
+  const lastTabErrorFatal = isLastStepErrorFatal(state, wizardID);
+
+  // Can always jump to first tab
+  result[visibleTabls[0]].canJumpTo = true;
+
+  // Can jump is previuse tab is valid and can jump to
+  for (let i = 1; i < visibleTabls.length; i++) {
+    const prev = result[visibleTabls[i - 1]];
+    const isPrevStepValid =
+      isSimpleView && isProviderImport
+        ? result[VMWizardTab.IMPORT_PROVIDERS].isValid
+        : prev?.isValid;
+
+    result[visibleTabls[i]].canJumpTo =
+      canJumpToAll || (!isLocked && !lastTabErrorFatal && prev?.canJumpTo && isPrevStepValid);
+  }
+
+  // Result tab is a special case
+  result[VMWizardTab.RESULT].canJumpTo =
+    lastTabErrorFatal || !result[VMWizardTab.RESULT].isValid == null;
```

Comment:
### 
```suggestion
    lastTabErrorFatal || !result[VMWizardTab.RESULT].isValid != null;
```

Reply:
we could add `isLocked` here as well for completeness

Reply:
added